frontend/src/types.ts

```diff
@@ -1432,6 +1432,20 @@ export interface SelectOption {
     label?: string
 }
 
+export enum FilterLogicalOperator {
+    And = 'AND',
+    Or = 'OR',
+}
+export interface AndOrPropertyFilter {
```

Comment:
The name here doesn't make sense to me. (and seems similar to the one below? Can't tell the difference using the names)

Perhaps call this `PropertyGroupFilter`?

and for the one below, can you explain where / why its used? 🤔

Reply:
yep!

Reply:
✅